go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -62,12 +65,28 @@ const (
 )
 
 var (
-	throttleThreshold         = flag.Duration("throttle_threshold", 1*time.Second, "Replication lag threshold for default lag throttling")
-	throttleTabletTypes       = flag.String("throttle_tablet_types", "replica", "Comma separated VTTablet types to be considered by the throttler. default: 'replica'. example: 'replica,rdonly'. 'replica' aways implicitly included")
-	throttleMetricQuery       = flag.String("throttle_metrics_query", "", "Override default heartbeat/lag metric. Use either `SELECT` (must return single row, single value) or `SHOW GLOBAL ... LIKE ...` queries. Set -throttle_metrics_threshold respectively.")
-	throttleMetricThreshold   = flag.Float64("throttle_metrics_threshold", math.MaxFloat64, "Override default throttle threshold, respective to -throttle_metrics_query")
-	throttlerCheckAsCheckSelf = flag.Bool("throttle_check_as_check_self", false, "Should throttler/check return a throttler/check-self result (changes throttler behavior for writes)")
+	// flag vars
+	throttleThreshold         = 1 * time.Second
+	throttleTabletTypes       = "replica"
+	throttleMetricQuery       string
+	throttleMetricThreshold   = math.MaxFloat64
+	throttlerCheckAsCheckSelf = false
+)
 
+func init() {
+	servenv.OnParseFor("vtcombo", registerThrottlerFlags)
+	servenv.OnParseFor("vttablet", registerThrottlerFlags)
+}
+
+func registerThrottlerFlags(fs *pflag.FlagSet) {
+	fs.DurationVar(&throttleThreshold, "throttle_threshold", throttleThreshold, "Replication lag threshold for default lag throttling")
+	fs.StringVar(&throttleTabletTypes, "throttle_tablet_types", throttleTabletTypes, "Comma separated VTTablet types to be considered by the throttler. default: 'replica'. example: 'replica,rdonly'. 'replica' aways implicitly included")
```

Comment:
you should see if you can use `fs.StringSliceVar` here and remove the splitting below

Reply:
This list supports multiple delimiters: comma, semi-colon or space. 
```
// SplitDelimitedList splits a given string by comma, semi-colon or space, and returns non-empty strings
func SplitDelimitedList(s string) (list []string) {
```